README.md

```diff
@@ -48,7 +48,8 @@ validate("/path/to/plugin/directory", run_all=True)
 to simulate the `--all` flag.
 
 ## Changelog
-
+* 2.8.0 - Update ChangelogValidator to validate plugin's version history with latest version number |
```

Comment:
Add a space between Changelog and the message. Check with MDL to make sure its formatted right

Reply:
@lmilby-r7 No need to check with MDL - that's part of the CI now :)